apps/sms/services/test/unit/mock_shared_worker.js

```diff
@@ -0,0 +1,5 @@
+/*exported MockSharedWorker */
+
+'use strict';
+
+function MockSharedWorker() {}
```

Comment:
I was hesitating to add such dumb mock (initially it was full with all SharedWorker methods, but in our own code we don't use them)